pkg/datastore/database.go

```diff
@@ -274,13 +295,22 @@ func (db *relationalDb) FindAllInTable(ctx context.Context, tableName string, re
 	return db.FindWithFilterInTable(ctx, tableName, record, records, pagination)
 }
 
+func (db *relationalDb) FindAllInTableIncludingSoftDeleted(ctx context.Context, tableName string, records interface{}, pagination *Pagination) error {
```

Comment:
FindAllInTableIncludingSoftDeletes?

Reply:
FindAllInTable() doesn't returns those that are already soft deleted. FindAllInTableIncludingSoftDeleted() includes those

Reply:
please have common method as the implementation is same  findAllInTable() with softDelete as param.